beebop/utils.py

```diff
@@ -94,3 +94,29 @@ def replace_filehashes(folder, filename_dict):
                 if f_key in line:
                     line = line.replace(f_key, f_value)
             print(line)
+
+
+def add_query_ref_status(fs, p_hash, filename_dict):
+    ET.register_namespace('', "http://graphml.graphdrawing.org/xmlns")
+    ET.register_namespace('xsi', "http://www.w3.org/2001/XMLSchema-instance")
+    # list of query filenames
+    query_names = list(filename_dict.values())
+    # list of all component graph filenames
+    file_list = []
+    for file in os.listdir(fs.output_network(p_hash)):
+        if fnmatch.fnmatch(file, 'network_component_*.graphml'):
+            file_list.append(file)
```

Comment:
```suggestion
    file_list = glob.glob("network_component_*.graphml")
```

With the glob module this should work (https://docs.python.org/3/library/glob.html#glob.glob)